custom_components/adjustable_bed/coordinator.py

```
@@ -1254,3 +1259,219 @@ def _notify_connection_state_change(self, connected: bool) -> None:
+                _LOGGER.debug("Position seek cancelled while waiting for lock")
+                if self._client is not None and self._client.is_connected:
+                    self._reset_disconnect_timer()
+                return
+
+            try:
+                # Clear cancel signal
+                self._cancel_command.clear()
+
+                if not await self.async_ensure_connected(reset_timer=False):
+                    _LOGGER.error("Cannot seek position: not connected to bed")
+                    raise ConnectionError("Not connected to bed")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)